packages/pari/pari.0.1/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Type-safe wrapper over the PARI library"
+maintainer: ["Julien Coolen <jtcoolen@pm.me>"]
+authors: ["Julien Coolen"]
+license: "GPL-2.0"
+homepage: "https://github.com/jtcoolen/ocaml-pari"
+bug-reports: "https://github.com/jtcoolen/ocaml-pari/issues"
+depends: [
+  "ocaml" {>= "4.10"}
+  "dune" {>= "3.7" & >= "3.7"}
+  "core" {>= "v0.14.0"}
+  "pari-bindings" {dev}
```

Comment:
```suggestion
  "pari-bindings" {= version}
```

The `version` version is a special value that means "the same version as this package". So it essentially means "this package (pari) depends on that package (pari-bindings), but that package (pari-bindings) can only be installed with the same (=) version as this package (pari)".

Reply:
Thanks! Applied your piece of advice in the upstream sources.